packages/webapp/components/Governance/ProposalCard.tsx

```diff
@@ -85,6 +85,7 @@ export const ProposalCard: FC<ProposalCardProps> = ({
           />
           {startBlock && endBlock && (
             <ProposalCountdown
+              // @ts-nocheck
```

Comment:
why "ts-nocheck"?

Reply:
Same as my previous response. Gnars types and Builder types do not always agree